### PR TITLE
[full-ci] Delete resharing tests

### PR DESCRIPTION
**Pull request by ScharfViktor**
- [x] deleted resharing test
- [x] check that re-sharing is not possible
- [x] change middleware https://github.com/owncloud/owncloud-test-middleware/pull/143
- [x] closed bugs:
- #2215 https://github.com/owncloud/enterprise/issues/3896 https://github.com/owncloud/ocis/issues/2213

Finding: 
- 500 while updating share expiration date without setting permission https://github.com/owncloud/ocis/pull/8704/files#diff-733a682a252e8a35d985692d58d7b793107aa0abe81eac800edd774e3415fc07R288-R292
- default permission is 31 when we don't set it https://github.com/owncloud/ocis/pull/8704/files#diff-71faa3d981b62b72aabeddca95a4aa975795e33f332bc6b19204f9cbd3604649R125-R127

e2e tests are blocked until it is merged https://github.com/owncloud/ocis/pull/8736

### CODE REVIEW

File: .drone.star [hunk 1]
```diff
@@ -18,7 +18,7 @@ OC_CI_PHP = "owncloudci/php:%s"
 OC_CI_WAIT_FOR = "owncloudci/wait-for:latest"
 OC_CS3_API_VALIDATOR = "owncloud/cs3api-validator:0.2.1"
 OC_LITMUS = "owncloudci/litmus:latest"
-OC_OC_TEST_MIDDLEWARE = "owncloud/owncloud-test-middleware:1.8.8"
+OC_OC_TEST_MIDDLEWARE = "owncloud/owncloud-test-middleware:latest"
```
Comment (individual-it) [inline .drone.star:21]: I would suggest to leave it pinned to a particular version

File: services/graph/pkg/service/v0/driveitems_test.go [hunk 2]
```diff
@@ -731,36 +731,37 @@ var _ = Describe("Driveitems", func() {
...
-			Expect(err).ToNot(HaveOccurred())
-
-			res := libregraph.Permission{}
-
-			err = json.Unmarshal(data, &res)
-			Expect(err).ToNot(HaveOccurred())
-			_, ok := res.GetRolesOk()
-			Expect(ok).To(BeTrue())
-		})
+		// that is resharing test. Please delete after disable resharing feature
```
Comment (individual-it) [inline services/graph/pkg/service/v0/driveitems_test.go:734]: do we want to merge the PR with that commented code?

Comment (ScharfViktor) [reply services/graph/pkg/service/v0/driveitems_test.go:734]: yes, we want to delete all resharing code and all resharing unit tests after merging this PR
cc @kobergj

File: .drone.star (hunk 1 above)
Comment (ScharfViktor) [reply .drone.star:21]: https://github.com/owncloud/owncloud-test-middleware/pull/143#issuecomment-2014516613

File: tests/acceptance/features/apiSpacesDavOperation/fileVersionByFileID.feature [hunk 3]
```diff
@@ -60,27 +59,29 @@ Feature: checking file versions using file id
...
 
   @issue-7738
-  Scenario: check the versions of a file after moving it to a shared folder of a project space shared with all permissions
+  Scenario: check the versions of a file after moving it to a shared folder inside a project space as manager
     Given user "Alice" has created a folder "testFolder" in space "Project1"
     And user "Alice" has created a share inside of space "Project1" with settings:
       | path      | testFolder |
       | shareWith | Brian      |
-      | role      | all        |
+    And user "Alice" has shared a space "Project1" with settings:
```
Comment (saw-jan) [inline tests/acceptance/features/apiSpacesDavOperation/fileVersionByFileID.feature:82]: do we need both steps here?

Comment (ScharfViktor) [reply tests/acceptance/features/apiSpacesDavOperation/fileVersionByFileID.feature:82]: Yes it needed. User should be member of the space(editor or manager) to see file version. 
If I delete this step -> test will be failed on 87 line (403 error)

Comment (saw-jan) [reply tests/acceptance/features/apiSpacesDavOperation/fileVersionByFileID.feature:82]: ahh, it didn't notice the sahre of testFolder

**Review by saw-jan — COMMENTED** [review]
Tests look good. :+1:

**Review by saw-jan — APPROVED** [review]

File: services/graph/pkg/service/v0/driveitems_test.go (hunk 2 above)
Comment (ScharfViktor) [reply services/graph/pkg/service/v0/driveitems_test.go:734]: The commented out code will be removed in following PRs 
see here https://github.com/owncloud/ocis/issues/8744

**Pull request merged**